engine/common/filesystem.c

```diff
@@ -251,8 +251,13 @@ static void listdirectory( stringlist_t *list, const char *path, qboolean lowerc
 	signed char *c;
 #ifdef _WIN32
 	char pattern[4096];
+#ifdef _WIN64
+	struct _finddatai64_t	n_file;
+	intptr_t		hFile;
+#else
```

Comment:
I think, you can add here #defines to not wrap everything in #ifdef ... #else ... #endif. That would be more readable.

Just a note for new engine: I suppose we need to define wrappers around directory iterating stuff and implement it in platform/win32 and platform/posix.